packages/firestore/test/integration/api/query.test.ts

```diff
@@ -2203,6 +2203,153 @@ apiDescribe('Queries', persistence => {
     }
   ).timeout('90s');
 
+  // TODO(b/291365820): Stop skipping this test when running against the
+  // Firestore emulator once the emulator is improved to include a bloom filter
+  // in the existence filter messages that it sends.
+  // eslint-disable-next-line no-restricted-properties
+  (USE_EMULATOR ? it.skip : it)(
+    'bloom filter can correctly handle updated and newly added documents',
+    async () => {
+      // Prepare the names and contents of the 100 documents to create.
+      const testDocs: { [key: string]: object } = {};
+      for (let i = 0; i < 100; i++) {
```

Comment:
Can you modify this test to use a smaller number of documents, like maybe 20 instead of 100? That would make the test execute more quickly and the large number of documents isn't strictly required to verify the bloom filter behavior.